CMakeLists.txt

```diff
@@ -70,5 +82,29 @@ set(VERSION_EDIT_WARNING "// Do not edit this file, it is automatically generate
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/Version.h.in ${CMAKE_CURRENT_BINARY_DIR}/src/Version.h)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/docker/post_build.sh.in ${CMAKE_CURRENT_BINARY_DIR}/post_build.sh)
 
+function(AddToListIfEnabled list enabled type)
+  if(${enabled})
+    list(APPEND ${list} ${type})
+  endif ()
+  #return(PROPAGATE ${list})
```

Comment:
Not enabled, but comment in PR sais it is

Reply:
The version of CMake in the docker container is too old for this. See the second commit in this PR.